ld-relay.go

```diff
@@ -45,10 +45,13 @@ type Config struct {
 	Environment map[string]*EnvConfig
 }
 
+type StatusEntry struct {
+	Status string `json:"status"`
+}
+
 var configFile string
 
 func main() {
```

Comment:
I got rid of the wait group, which allows the server to start before all of the environments have been initialized. This makes the whole system a little more resilient-- it can begin serving requests if (e.g.) one environment can't connect.